pkg/kube/controllers/quarkssecret/quarkssecret_controller.go

```diff
@@ -130,3 +193,9 @@ func listSecrets(ctx context.Context, client crc.Client, qsec *qsv1a1.QuarksSecr
 
 	return result, nil
 }
+
+func isUserCreatedSecret(secret *corev1.Secret) bool {
+	secretLabels := secret.GetLabels()
+	_, ok := secretLabels[qsv1a1.LabelKind]
```

Comment:
This is the label that is set to 'generated', so the opposite?

Reply:
yes. Added another check for the value too.